posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+
+[#return]
+=== Why did you want to return after your placement year, to start your career with IBM as a graduate?
+
+I wrote in detail above about what I worked on during my placement, what I learnt and also the incredible additional activities I was fortunate enough to attend during this year. The work at IBM is really varied – as a developer I expected to be coding frequently and learning a vast array of technology, but the role also encapsulated presenting, documentation, DevOps, reporting progress and conceiving ideas at meetings, dealing with customer support issues – and now, writing a blog post! 
+
+I think all of this was enough to have me champing at the bit to return as a graduate, but to extend on that – I met the most amazing people on my placement and that was my primary reason for being so keen to return. Of course, working with top talent presents the opportunity to learn an incredible amount, but I was somewhat surprised by the sheer approachability and kindness shown by these people – there were no stupid questions and despite worrying I would be taking up valuable time or distracting people from more important work I experienced nothing but hospitality and respect. In addition to this, I was matched with a truly fantastic mentor who has coached me throughout my career and been an invaluable source of support and guidance through both my placement year and beyond.
+ 
+Furthermore, there is a great social culture at IBM – as part of my internship we were encouraged to network with our peers which resulted in regular meet up activities such as a monthly pub quiz. Our office is in itself a community; it’s easy to get lost in conversation whilst replenishing beverages at the break room and we have regular team building events which have included a trip to the Bombay Sapphire distillery, regular barbecues and an annual Christmas party.
+ 
+Whilst COVID has impacted this somewhat the Hursley site is an incredible place to work. It is situated on over 100 acres of rural land in Hampshire. I utilised the beautiful surroundings to go for walks at lunch time and also the occasional run. Situated here is the Hursley clubhouse, which offers a wide range of sporting and social activities, as well as making IBM Hursley the only workplace in the UK which effectively has an on-site pub! There are site-wide events here, too, some of which are open to the public – for example there is the annual Hurfest event, annual fireworks, a recent archeological dig and it was my pleasure to attend the http://ukkubb.org[2021 UK Kubb Championship] here.
```

Comment:
There are site-wide events here, too, some of which are open to the public. For example, there is the annual Hurfest event, annual fireworks, a recent archeological dig and it was my pleasure to attend the http://ukkubb.org[2021 UK Kubb Championship] here.